### PR TITLE
added clisops package

### PR DESCRIPTION
**Pull request by cehbrecht**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by cehbrecht** [comment]
@Zeitsperre @ellesmith88 @agstephens Please could you confirm in this PR that you like to be a maintainer of this conda package.

**Comment by Zeitsperre** [comment]
Sounds good to me!

**Comment by ellesmith88** [comment]
Me too 👍

File: recipes/clisops/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 445324e32545468faaf8325c87393d0ce947d5f2be93c7319090edab8a7c4a9c
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<36]
```
Comment (scopatz) [inline recipes/clisops/meta.yaml:15]: please make into a single line `[win or py<36]`

Comment (cehbrecht) [reply recipes/clisops/meta.yaml]: done.

**Comment by agstephens** [comment]
I am happy to be  maintainer 👍

**Comment by cehbrecht** [comment]
@scopatz ready for merge :) Thanks for review.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**